src/test.rs

```diff
@@ -343,3 +344,20 @@ fn check_test_environment() {
     #[cfg(target_os = "linux")]
     fn test_iface() {}
 }
+
+#[test]
+fn test_get_network_interfaces_ips() {
+    test_iface_ips();
```

Comment:
I'm not convinced that this is the correct way to test this - it's entirely possible that the selected test interface will not have an IP address assigned to it. This said, the whole testing mechanism is a bit of a hack currently, so perhaps this is OK...